tests/lib_comfyui_tests/argv_conversion_test.py

```diff
@@ -1,6 +1,17 @@
+import os
 import unittest
-import importlib
-importlib.import_module('extensions.sd-webui-comfyui.tests.utils', 'utils').setup_test_env()
+import importlib.util
+
+# Get the module name and file name from the global directory path
+module_name = 'utils'
+file_name = 'utils.py'
+global_dir = os.path.abspath(os.path.join(os.path.dirname(os.path.dirname(os.path.realpath(__file__))), file_name))
+
+# Load the module from the file path
+spec = importlib.util.spec_from_file_location(module_name, global_dir)
+utils = importlib.util.module_from_spec(spec)
+spec.loader.exec_module(utils)
+utils.setup_test_env()
```

Comment:
do we really need to put all of this code here? If we have multiple test files this may not scale well. maybe it has a better place in utils.py?

Reply:
Was there a problem with `importlib.import_module('extensions.sd-webui-comfyui.tests.utils', 'utils').setup_test_env()`?

Reply:
Yes, pytest command would fail. I don't think this code is final, but that's the first iteration that managed to not make pytest error on module import. There's probably a better way, but I'm trying to figure out how to make the tests work